great_expectations/expectations/core/expect_table_column_count_to_be_between.py

```diff
@@ -110,7 +110,7 @@ class ExpectTableColumnCountToBeBetween(BatchExpectation):
         Passing Case:
             Input:
                 ExpectTableColumnCountToBeBetween(
-                    min_value=1
+                    min_value=1,
```

Comment:
Note for reviewer: I'm not sure where to preview this change nor am I confident it will have the desired effect. The expectations gallery seems to be outside the scope of the deploy preview for this PR https://deploy-preview-10560.docs.greatexpectations.io/

Is this the right way to make changes to expectation gallery entries?

Reply:
@JessSaavedra I bet you have an answer on this

Reply:
right now we don't have an automatized process to impact these changes in the expectations gallery, we run a script manually to regenerate the expectations-list.js file in the expectations gallery. After you've merged this PR we can do the update so that it gets reflected in the gallery :smile:

Reply:
Thanks @JessSaavedra ! I just merged this PR. Would you please run the script to update the gallery?